src/main/scala/ai/privado/exporter/SinkExporter.scala

```diff
@@ -55,8 +46,66 @@ class SinkExporter(cpg: Cpg) {
     convertSinkList(sinkTagList)
   }
 
-  def getProcessing: List[SinkProcessingModel] = {
+  def getProbableSinks: List[String] = {
+    var blackListedGroupIds: List[String] = List(
+      "<operator>.",
+      "<operators>.",
+      "iterator",
+      "assert",
+      "void.",
+      "java.",
+      "javax.",
+      "ANY.",
+      "android.",
+      "androidx.",
+      "okhttp3.",
+      "io.grpc.",
+      "io.netty.",
+      "org.apache.http.",
+      "org.json",
+      "org.junit.",
+      "org.apache.commons.",
+      "org.apache.tomcat",
+      "org.springframework.",
+      "com.fasterxml.jackson.",
+      "org.apache.hadoop.",
+      "org.apache.giraph.",
+      "org.javatuples.",
+      "org.antlr.",
+      "junit.framework.",
+      "io.reactivex.",
+      "com.bumptech.glide",
+      "com.google.protobuf",
+      "com.google.testing",
+      "com.google.common",
+      "<unresolvedNamespace>."
+    )
+    /** Get all the Methods which are tagged as SINKs  */
+    val taggedSinkMethods = cpg.tag.where(_.nameExact(Constants.catLevelOne).valueExact(CatLevelOne.SINKS.name)).call.l.map(i => i.methodFullName.split(":").head)
+
+    /** Get all the Methods which are external  */
+    val dependenciesTPs = cpg.method.external.l.map(i => i.fullName.split(":").head)
+
+    /** Actions:
+     * by excluding taggedSinkMethods
+     * using blackListedGroupIds
+     * transform method FullName close to groupIds
+     * remove duplicates
+     */
+    val filteredTPs = dependenciesTPs
+      .filter(str => !taggedSinkMethods.contains(str))
+      .filter((str) => {
+        val res = blackListedGroupIds.filter(blackListGroup => str.startsWith(blackListGroup))
+        (res.length == 0)
```

Comment:
we can write this as `blackListedGroupIds.filter(blackListGroup => str.startsWith(blackListGroup)).empty`